components/vocabularies/hpo/annotations/src/test/java/org/phenotips/vocabulary/internal/hpoannotations/PositivePhenotypeForDiseaseAnnotationsExtensionTest.java

```diff
@@ -122,7 +157,17 @@ public void getTargetVocabularyIds() throws Exception
     @Test
     public void getAnnotationSource() throws Exception
     {
-        // Assert.assertEquals(ANNOTATION_SOURCE, this.extension.getAnnotationSource());
+        final Provider<XWikiContext> xcontextProvider = this.mocker2.getInstance(XWikiContext.TYPE_PROVIDER);
```

Comment:
Same comments here - I think this test should be removed from here (and variables named better if it stays)